src/e3/anod/spec.py

```diff
@@ -203,6 +203,34 @@ class MyProduct(Anod):
     ExternalSourceBuilder = e3.anod.package.ExternalSourceBuilder
     ThirdPartySourceBuilder = e3.anod.package.ThirdPartySourceBuilder
 
+    def __new__(cls, *args: Any, **kwargs: Any) -> Any:
+        """Replace `method` by property when decorator is missing."""
+        if callable(cls.enable_name_generator):
```

Comment:
This could be factorized.
```python
should_be_property = ("readme_info", "build_space_name", ...)
for prop ins should_be_property:
    class_property = getattr(cls, prop)
    if callable(class_property):
        setattr(cls, prop, property(fget=class_property)
```
(or something like that)

Reply:
Done.